<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecvFile</class>
 <widget class="QDialog" name="RecvFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border:2px solid rgb(0, 122, 165);
border-radius:0px;
background-color: rgb(255, 255, 255);
}

QToolButton{
font: 12pt &quot;文泉驿等宽微米黑&quot;;
font: 14pt &quot;微软雅黑&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QLabel{
font: 12pt &quot;文泉驿等宽微米黑&quot;;
font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 122, 165);
}
QProgressBar {
font: 20pt &quot;文泉驿等宽微米黑&quot;;
font: 20pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border: 2px solid rgb(0, 125, 165);
border-radius: 8px;
text-align: center;
background-color: rgb(85, 170, 127);
background-color: rgb(0, 125, 165);
}

QProgressBar::chunk {
background-color: rgb(0, 255, 0);
}
</string>
  </property>
  <widget class="QProgressBar" name="serverProgressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>380</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbServerStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>430</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 122, 165);</string>
   </property>
   <property name="text">
    <string>文件接收</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pBtnClose">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>175</y>
     <width>110</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="../image.qrc">
     <normaloff>:/Image/close.png</normaloff>:/Image/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
